src/test/java/io/jenkins/plugins/analysis/core/model/IconLabelProviderTest.java

```diff
@@ -0,0 +1,51 @@
+package io.jenkins.plugins.analysis.core.model;
+
+import org.junit.jupiter.api.Test;
+
+import static io.jenkins.plugins.analysis.core.assertions.Assertions.*;
+
+/**
+ * Tests the class {@link IconLabelProvider}.
+ *
+ * @author Kezhi Xiong
+ */
+class IconLabelProviderTest {
+    private static final String iconId = "icon-id";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The constant name 'iconId' doesn't match '[A-Z][A-Z_0-9]*'](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5011839)